File: app/src/main/java/com/zeyad/rxredux/screens/user/list/viewHolders/EmptyViewHolder.java
package com.zeyad.rxredux.screens.user.list.viewHolders;

import com.zeyad.gadapter.GenericRecyclerViewAdapter;

import android.view.View;

/** @author zeyad on 11/29/16. */
public class EmptyViewHolder extends GenericRecyclerViewAdapter.ViewHolder {

    public EmptyViewHolder(View itemView) {
        super(itemView);
    }

    @Override
    public void bindData(Object data, boolean isItemSelected, int position, boolean isEnabled) {
    }
}

File: app/src/main/java/com/zeyad/rxredux/screens/user/list/viewHolders/EmptyViewHolder.kt
package com.zeyad.rxredux.screens.user.list.viewHolders

import android.view.View
import com.zeyad.gadapter.GenericRecyclerViewAdapter

class EmptyViewHolder(itemView: View) : GenericRecyclerViewAdapter.GenericViewHolder<Any>(itemView) {
    override fun bindData(data: Any, isItemSelected: Boolean, position: Int, isEnabled: Boolean) {}
}
